x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/utils.ts

```diff
@@ -102,3 +102,16 @@ export const checkIfSamePath = (pathA: ProcessorSelector, pathB: ProcessorSelect
   if (pathA.length !== pathB.length) return false;
   return pathA.join('.') === pathB.join('.');
 };
+
+/*
+ * Given a string it checks if its contents are a valid mustache template snippet.
+ *
+ * See: https://www.elastic.co/guide/en/elasticsearch/reference/master/ingest.html#template-snippets
+ */
+export const isTemplateSnippet = (str: string = '') => {
+  // Matches when:
+  //  * Starts with {{{
+  //  * Followed by all strings of length >= 1
+  //  * And ends with }}}
+  return /^{{{.+}}}$/.test(str);
```

Comment:
This allows strings with spaces such as: `{{{hello world}}}`. I figured I would use `.+` instead of `\S` (disallow all whitespaces) because the backend seems to allow spaces inside the template snippet.

Reply:
Mind adding the bit about whitespaces to the comment?